src/components/AppNavigation/ContactsSettings.vue

```diff
@@ -170,7 +170,7 @@ export default {
 .settings-new-addressbook >>> .icon-confirm {
 	min-height: 44px;
 	height: 44px;
-	border-color: var(--color-border-dark) !important;
+	border-color: black !important;
```

Comment:
I'm convinced this should use a css var, not a hard-coded value. The black border will be problematic with a bright theme.